### ISSUE TITLE
[BUG]: Unable to deserialize event

### ISSUE THREAD

**Issue by ilmax** [octokit/webhooks.net#297]
### What happened?

Exception thrown: 'System.Text.Json.JsonException' in Octokit.Webhooks.dll

It looks like the `DeserializeWebhookEvent` switch doesn't support the **deployment_review** event. 
Here's the GitHub headers sent for this event:

```
Request method: POST
Accept: */*
content-type: application/json
User-Agent: GitHub-Hookshot/10c8eb0
X-GitHub-Delivery: 1a99b8f0-1a4d-11ee-98f7-b32338c90d67
X-GitHub-Event: deployment_review
X-GitHub-Hook-ID: 406160204
X-GitHub-Hook-Installation-Target-ID: 308039
X-GitHub-Hook-Installation-Target-Type: integration
```

The configuration was setup as [follow ](https://docs.github.com/en/actions/deployment/protecting-deployments/creating-custom-deployment-protection-rules)

Let me know if I can help further here

### Versions

2.0.1

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by wolfy1339** [comment]
Can you create an issue in the `Octokit/webhooks` repo](https://github.com/octokit/webhooks), with some example payloads and I will take a look to try to make a best guess for a schema.

The more payloads that are different that you can provide the better

(You can anonymize the payloads, as long as the values are of the same type)

**Comment by ilmax** [comment]
Hello @wolfy1339 , it looks live this is due to #53 , Apparently an issue has been created already, see [this](https://github.com/octokit/webhooks/issues/555) and closed yet such event is still not implemented here.
Maybe unrelated but I'm still unable to find the documentation on the GitHub documentation [here](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads).

I was also looking at the code to simplify the creation of unsupported webhooks by the library and there're some limitation now (i.e. the class `WebhookConverter<T>` only look for webhooks in the `Octokit.Webhooks` assembly so if you implement your custom payload you need to implement also the deserialization, few attributes are also internal)

I made a few changes to simplify the extension process (i.e. support webhooks that are not yet supported by the library) and if you're interested I can submit a PR here, just let me know

**Comment by wolfy1339** [comment]
I reopened the issue. Feel free to post some example payloads there

**Comment by JamieMagee** [comment]
Looks like this was resolved in #329
